### DISCUSSION TITLE
Get installments with custom amount

### DISCUSSION THREAD

**Discussion by M4teus2020** [mercadopago/sdk-js#180] [Q&A] [answered]
Oi tudo bem? queria saber se tem alguma forma de eu recarregar as parcelas com um valor diferente do q esta no amount da criação do cardForm do checkout transparente

Ou se tiver alguma forma de atualizar o amount pra quando e forçar recarregar com o getInstallments.

Não consegui encontrar em lugar nenhum nem no discord do mercado pago nem aqui, muito menos na documentação do site

**Comment by lucmantovani** [comment]
Olá @M4teus2020!
Infelizmente o Card Form não tem essa funcionalidade
Nesse caso o ideal seria implementar uma integração utilizando Core Methods, pois assim você tem o controle completo sobre o fluxo do checkout

**Reply by M4teus2020** [reply]
Muito Obrigado, irei tentar fazer a implementação dessa forma

**Comment by ghost** [comment, marked as answer]
Bom dia @M4teus2020 , tudo bem e você?
A sugestão do @lucmantovani é válida, mas para facilitar sua integração, gostaria de recomendar nosso produto Checkout Bricks, ele já vem com essa funcionalidade de atualizar o amount, inclusive busca as novas parcelas automaticamente para você 😃 .
Os produtos que contam com essa funcionalidade são o Card Payment Brick e o Payment Brick. Por ser uma funcionalidade nova, ainda não temos documentado no site do Mercado Pago Developers (estamos trabalhando nisso), mas deixo os links das documentações técnicas no Github:
1. Payment Brick: https://github.com/mercadopago/sdk-js/blob/main/API/bricks/payment.md#brick-controllerupdate
2. Card Payment Brick: https://github.com/mercadopago/sdk-js/blob/main/API/bricks/card-payment.md#brick-controllerupdate

Particularmente, recomendo a integração com Payment Brick por ser mais completo, contando com pagamentos via cartão, Pix, boleto, lotérica e Conta Mercado Pago.
Deixo aqui as demonstrações dos dois produtos para você testar:
1. Payment Brick: https://www.mercadopago.com.br/developers/pt/live-demo/payment-brick
2. Card Payment Brick: https://www.mercadopago.com.br/developers/pt/live-demo/card-payment-brick

Qualquer dúvida, estamos à disposição, só mandar mensagem nesta mesma discussion. Muito obrigado!

**Reply by ghost** [reply]
@M4teus2020 atualização: agora temos a documentação do método `update` no site do Mercado Pago Developers:
1. Payment Brick: https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/payment-brick/additional-customization/update-data
2. Card Payment Brick: https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/card-payment-brick/additional-customization/update-data

**Reply by M4teus2020** [reply]
Muito Obrigado, Vou tentar primeiro pelo Método Core, se não conseguir terei que fazer dessa forma.

Tamo junto!
